<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMuseum</class>
 <widget class="QDialog" name="AddMuseum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next Condensed</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Add Museum&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mListButton">
      <property name="text">
       <string>Museum list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="itemsViewedButton">
      <property name="text">
       <string>Items Viewed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myCommentsButton">
      <property name="text">
       <string>My Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountButton">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logoMM_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>771</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>New Museum</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Museum Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="museumName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Museum Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="museumDescription"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Museum Type</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="museumType"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Museum</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
